WordPress/src/main/java/org/wordpress/android/ui/main/SitePickerActivity.java

```diff
@@ -205,6 +204,17 @@ protected void onStart() {
 
     @SuppressWarnings("unused")
     @Subscribe(threadMode = ThreadMode.MAIN)
+    public void onSiteRemoved(OnSiteRemoved event) {
+        if (!event.isError()) {
+            getAdapter().loadSites();
+        } else {
+            // shouldn't happen
```

Comment:
Ah right, I missed that. We should definitely localize the toast message here: `ToastUtils.showToast(this, "Error removing site, try again later");`